packages/gestalt-core/build.js

```diff
@@ -57,7 +57,14 @@ const cssModules = (options = {}) => {
         scopeNames[hash] = classnameBuilder.getMinifiedClassname(hash);
       }
 
-      return scopeNames[hash];
+      // if it's dev mode, also append the full class name for debugging
+      const isDevMode = process.argv.includes('-w') || process.argv.includes('--watch');
```

Comment:
can you explain / add a comment for this?

process.argv.includes('-w') || process.argv.includes('--watch');

Reply:
@rlingineni